ts-eks/test/klotho.yaml

```diff
@@ -4,63 +4,4 @@ path: .
 out_dir: compiled
 defaults:
     execution_unit:
-        type: lambda
-        pulumi_params_by_type:
-            fargate:
-                cpu: 256
-                memory: 512
-            lambda:
-                memorySize: 512
-                timeout: 180
-    static_unit:
-        type: s3
-        pulumi_params_by_type:
-            s3:
-                cloudFrontEnabled: true
-    expose:
-        type: apigateway
-    persist:
-        kv:
-            type: dynamodb
-        fs:
-            type: s3
-        secret:
-            type: s3
-        orm:
-            type: rds_postgres
-            pulumi_params_by_type:
-                rds_postgres:
-                    allocatedStorage: 20
-                    engineVersion: "13.7"
-                    instanceClass: db.t4g.micro
-                    skipFinalSnapshot: true
-        redis:
-            type: elasticache
-            pulumi_params_by_type:
-                elasticache:
-                    nodeType: cache.t3.micro
-                    numCacheNodes: 1
-    pubsub:
-        type: sns
-execution_units:
-    main:
         type: eks
```

Comment:
if this is for pro, dont we want multiple exec units?

Reply:
Yeah, this is changing the default so it should apply to all

Reply:
ah i see, will approve for now, but we want to test both fargate and node types as well as other configurations, so well likely want to refactor this in the future to mirror what we had in the config before the pro split

Reply:
Yeah, I think what we'll _really_ want is the ability to have multiple config files, each turning into its own pair of tests.